layouts/partials/comments/staticman/form.html

```diff
@@ -8,6 +8,9 @@
 
 {{- $api := ( print $apiEndpoint "/" $username "/" $repo "/" $branch ) -}}
 
+{{- $staticman := ( partialCached "comments/staticman/GetStaticmanYML" . ) -}}
```

Comment:
If you want to avoid duplicate input, then `branch` should be treated equally as `reCaptcha` parameters.

```
{{- $branch := $staticman.branch -}}
```

Reply:
Personally, I wouldn't go for this alone, but in the spirit of your Hugo cache approach, I suggested this little change.